pycalculator.py

```diff
@@ -0,0 +1,42 @@
+import pickle
+
+
+def div(dividend: int, divisor: int):
+    """
+    Calculate the division of two numbers.
+    The dividend is divided by the divisor to get a quotient.
+
+    :return: the quotient.
+    """
+    return dividend / divisor
+
+
+def sum(num1: int, num2: int):
+    """
+    Calculate the sum of two numbers.
+
+    :return: The sum of num1 and num2.
+    """
+    return num1 + num2
+
+
+def save_number(num: int, path: str):
+    """
+    Saves the number in the path as a pickle file.
+
+    :param num: The number to save.
+    :param path: Where to save the number.
+    """
+    with open(path, 'wb') as num_file:
+        pickle.dump(num, num_file)
```

Comment:
Why `pickle` instead of a simple text file?
It's only a number, and `pickle` is a pretty big overkill here..